Dockerfile

```diff
@@ -190,6 +190,22 @@ RUN set -ex; \
         done; \
     fi
 
+# pgai is an extension for artificial intelligence workloads
+ARG PGAI_VERSION
+RUN set -ex; \
+    if [ -n "${PGAI_VERSION}" ]; then \
+        git clone https://github.com/timescale/pgai.git /build/pgai; \
+        cd /build/pgai; \
+        git checkout "${PGAI_VERSION}"; \
+        git reset HEAD --hard; \
+        for pg in ${PG_VERSIONS}; do \
+            cp /build/pgai/ai--*.sql "$(/usr/lib/postgresql/${pg}/bin/pg_config --sharedir)/extension/"; \
+            cp /build/pgai/ai.control "$(/usr/lib/postgresql/${pg}/bin/pg_config --sharedir)/extension/"; \
+        done; \
+        apt-get install -y python3-pip; \
```

Comment:
If you rebase on master, `python3-pip` is already installed on line 150, so you could drop that.